### DISCUSSION TITLE
Regarding GSoC 2024

### DISCUSSION THREAD

**Discussion by RiriPiri** [dipy/dipy#2951] [#GSoC/GSoD]
Hello Sir/ Madam, I wanted to ask about the estimated time when the GSoC 2024 project ideas will be posted. I am a third year Biomedical Engineering student and have worked in the prediction and detection of diseases using Python under my professors. My paper is under review for the same. Thank you.

**Comment by pjsjongsung** [comment]
We do not have a timeline on when the project ideas will be posted. But we hope to have another year of GSoC next year under PSF.  Please keep an eye on the Wiki page in the future!

**Reply by RiriPiri** [reply]
Okay. Thank you, Sir.
